### PR TITLE
House keeping: Remove legacy office screen

### PR DESCRIPTION
**Pull request by alesshh**
## Context
This PR removes legacy code from old matrix office screen.

All should work, without brakes.
Besides that, we can redirect logged users on the backend, keeping the frontend simple.
On a next PR I will refactor login page to morpheus.

fix: https://github.com/ResultadosDigitais/matrix/issues/225

### CODE REVIEW

File: backend/app/views/morpheus.ejs [hunk 1]
```diff
@@ -15,7 +15,7 @@
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
     />
     <%- include('favicons'); %>
-    <title>Matrix - Morpheus</title>
```
Comment (juliemar) [inline backend/app/views/morpheus.ejs]: What do you think to maintain the #matrix - morpheus?

File: backend/app/views/morpheus.ejs [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
+    <title>Matrix</title>
```
Comment (juliemar) [inline backend/app/views/morpheus.ejs]:
```suggestion
    <title>#matrix</title>
```

File: frontend/package.json [hunk 3]
```diff
@@ -10,14 +10,11 @@
   "dependencies": {
     "@material-ui/core": "^4.4.1",
     "@material-ui/icons": "^4.4.1",
-    "@sentry/browser": "^5.6.3",
     "axios": "^0.19.0",
-    "bootstrap": "^4.3.1",
+    "bootstrap": "^4.4.1",
```
Comment (juliemar) [inline frontend/package.json]: Is It Necessary to maintain bootstrap? I don't know Where are we continue using :)

Comment (alesshh) [reply frontend/package.json]: It's used on the login screen. I will 🔪 on a login screen refactor.

File: backend/app/views/morpheus.ejs (hunk 1 above)
Comment (alesshh) [reply backend/app/views/morpheus.ejs]: Better!

**Pull request merged**